test/App.fs

```diff
@@ -21,8 +21,8 @@ type Msg =
   | CountUpdated of float option
 
 let init() : Model * Cmd<Msg> =
-  { I18nEn = I18n(Fable.Core.JsInterop.importDefault(__SOURCE_DIRECTORY__ + "/strings.en.json"), "en")
-    I18nFr = I18n(Fable.Core.JsInterop.importDefault(__SOURCE_DIRECTORY__ + "/strings.fr.json"), "fr")
+  { I18nEn = I18n(Fable.Core.JsInterop.importDefault(__SOURCE_DIRECTORY__ + "/test/strings.en.json"), "en")
```

Comment:
dotnet fsi build.fsx start is failing with `.\test\App.fs(24,18): (24,100) error FABLE: Expecting string literal`.
You can try to remove the string concatenation here.
Has we are trying to load sibling files, `Fable.Core.JsInterop.importDefault("./strings.en.json")` should work